packages/graph/hash_graph/README.md

```diff
@@ -2,38 +2,38 @@
 
 ## Run the Graph
 
-The easiest way to run the Graph API is to use docker. Either invoke docker directly:
+The easiest way to run the Graph API is to use docker through `cargo make`:
 
 ```shell
-DOCKER_BUILDKIT=1 docker build --tag graph --file ../deployment/graph/Dockerfile .
-docker run --rm --init --network host --name graph --env RUST_LOG=trace --detach graph
+cargo make build-docker --profile production
+cargo make graph-up
 ```
 
-or by using `cargo make`:
+or by using `yarn`
 
 ```shell
-cargo make build-docker --profile production
-cargo make docker-up
+DOCKER_BUILDKIT=1 yarn external-services build graph
+yarn external-services up --detach graph
```

Comment:
Not sure if I'm missing something, but why are we mentioning this script here? It was my understanding we would treat the `yarn external-services up` docker-compose configuration as the configuration for setting up the developer environment of the HASH workspace, not the developer environment of the HASH graph.

Also note that this script has to be run in the root of the repository, whereas the script it's replacing would've been run in this directory.

Reply:
It was also my understanding that longterm we want to move the `yarn external-services` and related workspace scripts into `packages/hash`, at which point I don't think you could call it from this directory anymore.

Reply:
As discussed off-GH, we remove the entire section on running `graph` with docker. We expect a user for `graph` to use `cargo run`. For the HASH package, the external services are used.